lib_test/Test_Frenetic_NetKAT.ml

```diff
@@ -1,575 +1,5 @@
-  check gen_pol_3 prop_compile_ok
-
-let%test "quickcheck ka-seq-dist-r compiler" =
-  let prop_compile_ok (p, q, r, pkt) =
-    compare_compiler_output
-      (Seq (Union(p, q), r)) (Union (Seq(p, r), Seq(q, r))) pkt in
-  check gen_pol_3 prop_compile_ok
-
-let%test "quickcheck ka-seq-dist-r eval" =
-  let prop_compile_ok (p, q, r, pkt) =
-    compare_eval_output
-      (Seq (Union(p, q), r)) (Union (Seq(p, r), Seq(q, r))) pkt in
-  check gen_pol_3 prop_compile_ok
-
-let%test "quickcheck ka-zero-seq compiler" =
-  let prop_compile_ok (pol, pkt) =
-    compare_compiler_output drop (Seq(drop, pol)) pkt in
-  check gen_pol_1 prop_compile_ok
-
-let%test "quickcheck ka-zero-seq eval" =
-  let prop_compile_ok (pol, pkt) =
-    compare_eval_output drop (Seq(drop, pol)) pkt in
-  check gen_pol_1 prop_compile_ok
-
-let%test "quickcheck ka-seq-zero compiler" =
-  let prop_compile_ok (pol, pkt) =
-    compare_compiler_output drop (Seq(pol, drop)) pkt in
-  check gen_pol_1 prop_compile_ok
-
-let%test "quickcheck ka-seq-zero eval" =
-  let prop_compile_ok (pol, pkt) =
-    compare_eval_output drop (Seq(pol, drop)) pkt in
-  check gen_pol_1 prop_compile_ok
-
-let%test "quickcheck ka-unroll-l compiler" =
-  let prop_compile_ok (pol, pkt) =
-    compare_compiler_output (Star pol) (Union(id, Seq(pol, Star pol))) pkt in
-  check gen_pol_1 prop_compile_ok
-
-let%test "quickcheck ka-unroll-l eval" =
-  let prop_compile_ok (pol, pkt) =
-    compare_eval_output (Star pol) (Union(id, Seq(pol, Star pol))) pkt in
-  check gen_pol_1 prop_compile_ok
-
-let%test "quickcheck ka-lfp-l compiler" =
-  let prop_compile_ok (p, q, r, pkt) =
-    not (compare_compiler_output (Union(Union(q, Seq (p, r)), r)) r pkt)
-    ||  (compare_compiler_output (Union(Seq(Star p, q), r)) r pkt) in
-  check gen_pol_3 prop_compile_ok
-
-let%test "quickcheck ka-lfp-l eval" =
-  let prop_compile_ok (p, q, r, pkt) =
-    not (compare_eval_output (Union(Union(q, Seq (p, r)), r)) r pkt)
-    ||  (compare_eval_output (Union(Seq(Star p, q), r)) r pkt) in
-  check gen_pol_3 prop_compile_ok
-
-let%test "quickcheck ka-unroll-r compiler" =
-  let prop_compile_ok (pol, pkt) =
-    compare_compiler_output (Star pol) (Union(id, Seq(Star pol, pol))) pkt in
-  check gen_pol_1 prop_compile_ok
-
-let%test "quickcheck ka-unroll-r eval" =
-  let prop_compile_ok (pol, pkt) =
-    compare_eval_output (Star pol) (Union(id, Seq(Star pol, pol))) pkt in
-  check gen_pol_1 prop_compile_ok
-
-let%test "quickcheck ka-lfp-r compiler" =
-  let prop_compile_ok (p, q, r, pkt) =
-    not (compare_compiler_output (Union(Union(p, Seq (q, r)), q)) q pkt)
-    ||  (compare_compiler_output (Union(Seq(p, Star r), q)) q pkt) in
-  check gen_pol_3 prop_compile_ok
-
-let%test "quickcheck ka-lfp-r eval" =
-  let prop_compile_ok (p, q, r, pkt) =
-    not (compare_eval_output (Union(Union(p, Seq (q, r)), q)) q pkt)
-    ||  (compare_eval_output (Union(Seq(p, Star r), q)) q pkt) in
-  check gen_pol_3 prop_compile_ok
```

Comment:
My mistake.  They seemed to be more appropriate in Test_Frenetic_Compiler, so I was moving them from Test_Frenetic_NetKat, but dropped them on the way over.  Will fix...